src/Modules/Manage/Dnn.PersonaBar.Users.Tests/AddRolesUnitTests.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Moq;
+using NUnit.Framework;
+using DotNetNuke.Entities.Users;
+using DotNetNuke.Security.Roles;
+using Dnn.PersonaBar.Library.Prompt;
+using Dnn.PersonaBar.Roles.Components;
+using Dnn.PersonaBar.Users.Components;
+using Dnn.PersonaBar.Users.Components.Prompt.Commands;
+using Dnn.PersonaBar.Library.Prompt.Models;
+
+namespace Dnn.PersonaBar.Users.Tests
+{
+    [TestFixture]
+    public class AddRolesUnitTests : CommandTests
+    {
+        private Mock<IUserValidator> _userValidatorMock;
+        private Mock<IUsersController> _usersControllerMock;
+        private Mock<IRolesController> _rolesControllerMock;
+
+        [SetUp]
+        public void RunBeforeEveryTest()
+        {
+            _userValidatorMock = new Mock<IUserValidator>();
+            _usersControllerMock = new Mock<IUsersController>();
+            _rolesControllerMock = new Mock<IRolesController>();
+            Reset();
```

Comment:
This isn't ideal, because someone writing tests must be aware of this method. Instead I suggest creating a public setup method in base class and an abstract setup method for the inheriting fixture

```c#
public class CommandTests
{
   [SetUp]
   public void Setup()
   {
       // what Reset() does
       ChildSetup();
   }

   protected abstract void ChildSetup();
}

public class AddRolesUnitTests : CommandTests
{
   protected override void ChildSetup()
   {
      _userValidatorMock = new Mock<IUserValidator>(); // etc
   }
}
```